app/main/views/services.py

```diff
@@ -41,6 +43,10 @@ def list_services():
 
     services = Service.query.filter(
         Service.framework.has(Framework.expired == false())
+    ).order_by(
```

Comment:
I'm not sure if that's true (at least not due to anything Pagination does). The next page link in the API response only has `?page=n` and the `.paginate` method doesn't add an `order_by`.